docs/pages/operations-guide.md

```diff
@@ -371,361 +383,285 @@ Given the above configuration, let's assume `10.0.10.8` is being decommissioned
+```
 
-As of this writing, the best and fastest way to view and download logs is via the Mesos UI at `<dcos-url>/mesos`. On the Mesos front page you will see two lists: A list of currently running tasks, followed by a list of completed tasks (whether successful or failed).
+Once we know the configuration is good, it should be added to our source control for tracking.
 
-[<img src="img/ops-guide-all-tasks.png" alt="mesos frontpage showing all tasks in the cluster" width="400"/>](img/ops-guide-all-tasks.png)
+## Updating service configuration
 
-The `Sandbox` link for one of these tasks shows a list of files from within the task itself. For example, here's a sandbox view of a `broker-2` task from the above list:
+Above, we described how a configuration update (including updating the version of the service) is handled. Now we will quickly show the steps to perform such an update.
 
-[<img src="img/ops-guide-task-sandbox.png" alt="contents of a task sandbox" width="400"/>](img/ops-guide-task-sandbox.png)
+Configuration updates are performed by updating the process environment of the Scheduler. Once restarted, the Scheduler will observe this change and re-deploy nodes as described in ][Reconfiguration](#Reconfiguration).
 
-If the task is based on a Docker image, this list will only show the contents of `/mnt/sandbox`, and not the rest of the filesystem. If you need to view filesystem contents outside of this directory, you will need to use `dcos task exec` or `nsenter` as described below under [Running commands within containers](#running-commands-within-containers).
+### Enterprise DC/OS 1.10
 
-In the above task list there are multiple services installed, resulting in a pretty large list. The list can be filtered using the text box at the upper right, but there may be duplicate names across services. For example there are two instances of `confluent-kafka` and they're each running a `broker-0`. As the cluster grows, this confusion gets proportionally worse. We want to limit the task list to only the tasks that are relevant to the service being diagnosed. To do this, click "Frameworks" on the upper left to see a list of all the installed frameworks (mapping to our services):
+Enterprise DC/OS 1.10 introduces a convenient command line option that allows for easier updates to a service's configuration, as well as allowing users to inspect the status of an update, to pause and resume updates and to restart or complete steps if necessary. 
 
-[<img src="img/ops-guide-frameworks-list.png" alt="listing of frameworks installed to the cluster" width="400"/>](img/ops-guide-frameworks-list.png)
+#### Prerequisites
 
-We then need to decide which framework to select from this list. This depends on what task we want to view:
++ Enterprise DC/OS 1.10 or newer.
++ A DC/OS SDK-based service with a version greater than 2.0.0-x
++ [The DC/OS CLI](https://docs.mesosphere.com/1.9/cli/install/) installed and available
```

Comment:
that version branch isn't live yet, but since this is information specific to 1.10, it seems right to have the 1.10 doc link.